native/chat/chat.react.js

```diff
@@ -148,17 +153,37 @@ export type ChatNavigationProp<
   RouteName: $Keys<ChatParamList> = $Keys<ChatParamList>,
 > = ChatRouterNavigationProp<ScreenParamList, RouteName>;
 
+export type ChatTopTabsNavigationProp<
+  RouteName: $Keys<ChatTopTabsParamList> = $Keys<ChatTopTabsParamList>,
+> = ChatRouterNavigationProp<ScreenParamList, RouteName>;
+
 const Chat = createChatNavigator<
   ScreenParamList,
   ChatParamList,
   ChatNavigationProp<>,
 >();
+
+const ChatThreadsTopTab = createMaterialTopTabNavigator();
+const ChatThreadsComponent = () => {
+  return (
+    <ChatThreadsTopTab.Navigator>
+      <ChatThreadsTopTab.Screen
+        name={HomeChatThreadListRouteName}
+        component={HomeChatThreadList}
```

Comment:
Reminder - we want to explicitly specify the title/labels here. Right now it's defaulting to using the route name, but that's meant to be an internal detail instead of a UI label

Reply:
I'll do that in next PR, didn't want to put everything here